### ISSUE TITLE
Force HTP/LTP editor broken

### ISSUE THREAD

**Issue by janosvitok** [mcallegari/qlcplus#854]
[Forum thread](http://www.qlcplus.org/forum/viewtopic.php?f=20&t=9985)

[Channel Selection](https://github.com/mcallegari/qlcplus/blob/master/ui/src/channelsselection.cpp#L156) reads current state of ForceLTP/ForceHTP from universe flags, which is not always OK. It should read the value from fixture's ForceHTP/ForceLTP settings and channel group.

Perhaps the translation of group to channel capabilities (Intensity -> HTP|Intensity, others -> LTP) could be extracted to a common method somewhere.

**Comment by mcallegari** [comment]
@janosvitok how did you reproduce this ?

**Comment by janosvitok** [comment]
Steps to reproduce:
1. add 2 channels of dimmers
2. change channel 1 to LTP
3. save
4. reopen
5. change channel 2 to LTP (now both are LTP)
6. save
7. reopen
8. open channel modifiers, now channel 1 is HTP (LTP setting is lost).

Key thing here is that when you modify freshly opened file and change one channel, the rest of changes are lost, because they are read from Universe instead of Fixture (most probably when HTP/LTP setting in Universe (effective) is the same as HTP/LTP setting in the dialog, the setting is not saved).

In other words, when the forced setting is already applied to Universe, the code assumes the value is not changed from default.

**Comment by mcallegari** [comment]
I did the above and couldn't reproduce with the current sources.
BUT
I closed QLC+ on every save/reopen. Maybe that's what makes the difference ?

**Comment by janosvitok** [comment]
I tested with 4.11.0 windows version. I'll try latest appveyor.

**Comment by mcallegari** [comment]
@janosvitok ...actually there were like 800 errors in 10 lines of code...
As soon as AppVeyor is done, please test. Should be OK in all cases now

**Comment by janosvitok** [comment]
Quick retest with 4.11.1.131.exe OK.
